src/main/java/ome/services/fulltext/BridgeHelper.java

```diff
@@ -102,6 +103,66 @@ public abstract void set(final String name, final Object value,
      *            this value is null, then the "value" will only be added to the
      *            {@link #COMBINED} field.
      * @param value
+     *            Value which has been parsed out for this field. If null, then
+     *            this is a no-op.
+     * @param opts
+     *            LuceneOptions, passed in from the runtime. If overriding on
+     *            the interface values is required, see {@link SimpleLuceneOptions}
+     */
+    protected void addIfNotNull(Document d, String field, String value, LuceneOptions opts) {
+        if (value != null) {
+            add(d, field, value, opts);
+        }
+    }
+
+    /**
+     * Helper method which takes the parameters from the
+     * {@link #set(String, Object, Document, LuceneOptions)}
+     * method (possibly modified) as well as the {@link IEnum} value
+     * which should be added to the index, and adds two fields. One with the
+     * given field name and another to the {@link #COMBINED} field which is the
+     * default search provided to users. In addition to storing the value as is,
+     * another {@link Field} will be added for both the named and
+     * {@link #COMBINED} cases using a {@link StringReader} to allow Lucene to
+     * tokenize the {@link String}.
+     *
+     * @param d
+     *            Document as passed to the set method. Do not modify.
+     * @param field
+     *            Field name which probably <em/>should</em> be modified. If
+     *            this value is null, then the "value" will only be added to the
+     *            {@link #COMBINED} field.
+     * @param value
+     *            {@link IEnum} whose {@link IEnum#getValue()} method will be alled.
```

Comment:
called

Reply:
*cough* https://thenextweb.com/dd/2018/10/16/github-launches-suggested-changes-to-make-it-easier-to-collaborate-within-pull-requests/ *cough*